codec-memcache/src/main/java/io/netty/handler/codec/memcache/binary/AbstractBinaryMemcacheMessage.java

```diff
@@ -66,7 +66,7 @@ public ByteBuf extras() {
     }
 
     @Override
-    public BinaryMemcacheMessage setKey(String key) {
+    public BinaryMemcacheMessage setKey(ByteBuf key) {
         this.key = key;
```

Comment:
@windie @daschl I wonder if we need to call `release()` on the previous key if there was one.